docs/source/quickstart.rst

```diff
@@ -275,3 +284,164 @@ At this point OHIF can be used to annotate the data in the DICOM server via the
 .. note::
 
   Here, user may also perform annotation using 3DSlicer by following the same instructions as in section :ref:`DeepEdit Annotation in 3DSlicer`.
+
+
+
+.. _Bundle:
+
+=============================================
+Tutorial 2 - Bundle App Tutorial and Use Cases
+=============================================
+
+Introduction:
+===============
+
+Customized Networks for MONAI Label
+------------------------------------
+
+This tutorial introduces the usage of the Bundle app in MONAILabel  - **monaibundle**.
+
+The Bundle App empowers MONAILabel with customized models, pre- and post-processing, and any anatomies for labeling tasks.
+The Bundle App supports various training/inference frameworks with `Model-Zoo <https://github.com/Project-MONAI/model-zoo>`_
+
+Highlights and Features:
+
+  * Supporting customized models and networks such as SwinUNETR, AutoML, etc.
+  * Advancing heterogeneous dataset (e.g., CT, MRI, Pathology, etc) with corresponding pre- and post-processing modules.
+  * Ready-to-Use inference of hundreds of anatomies (e.g., multi-organ abdominal segmentation, whole-brain segmentation) with trained model checkpoints.
+  * Deploying robust interactive labeling tools such as DeepEdit.
+
+.. _Model Zoo for MONAI Label:
+
+Model Zoo for MONAI Label
+-----------------------------
+
+MONAI Model Zoo hosts a collection of medical imaging models in the MONAI Bundle format.
+All source code of models (bundles) are tracked in models/, and for each distinct version of a bundle,
```

Comment:
We should not expect users to check the source code of `model-zoo` repo, please use the release link of models: https://github.com/Project-MONAI/model-zoo/releases/tag/hosting_storage_v1

Thanks.

Reply:
yes, good suggestion. Changed the Model zoo link to the release link of models.